project1.1.js

```diff
@@ -0,0 +1,141 @@
+
+/*-----------Boiler Plate\readline standards---------*/
+
+const readline = require('readline');
+const rl = readline.createInterface(process.stdin, process.stdout);
+
+function ask(questionText) {
+    return new Promise((resolve, reject) => {
+        rl.question(questionText, resolve);
+    });
+}
+
+/*----------------Global-----------*/
+
+let min = 1;
+let max = 100;
```

Comment:
Since you are reassigning the value with the await ask function, you dont have to assign a value to min/max but you are right to declare it globally. But you could have a `let min;` without a value being assigned